thunder/clang/__init__.py

```diff
@@ -598,7 +598,18 @@ def _basic_indexing(a: TensorLike, /, key) -> TensorLike:
     if key is None or isinstance(key, (Number, NumberProxy, slice, EllipsisType)):
         key = (key,)
 
+    _key = []
```

Comment:
Part of the reason I'd like to separate this PR more into updates to advanced indexing and then support for combined indexing is that I'd like to better understand updating basic indexing to support the combined indexing. I think a discussion of approaches for combined indexing support would be really interesting.

Reply:
My intention here was to convert any "advanced indices" with slice(None,None,None) so that basic indexing could "skip" it (i.e. no effect to basic indexing). To update that advanced key, I needed a variable to keep those basic keys + converted advanced keys.